packages/search/package.json

```diff
@@ -17,14 +17,14 @@
     "rollup": "../../node_modules/rollup/bin/rollup -c rollup.config.js",
     "build:ts": "tsc -p tsconfig.build.json",
     "clean:ts": "../../node_modules/rimraf/bin.js ts-out",
-    "build ": "npm run build:ts && npm run rollup && npm run clean:ts"
+    "build": "npm run build:ts && npm run rollup && npm run clean:ts",
+    "test": "../../node_modules/mocha/bin/mocha --opts ../../default.opts \"test/*.spec.ts\""
   },
   "peerDependencies": {
     "@craftercms/models": "0.0.0-PLACEHOLDER",
     "@craftercms/utils": "0.0.0-PLACEHOLDER",
-    "rxjs": "^6.2.0"
-  },
-  "dependencies": {
-    "uuid": "^3.2.1"
+    "rxjs": "^6.2.0",
+    "uuid": "^3.3.2",
+    "chai": "^4.1.2"
```

Comment:
`chai` is used exclusively for testing, correct? dependencies on the package should be _functional_ dependencies. all other go on devDeps of the root package.json